### ISSUE TITLE
[low] Chrome + Talkback: Screen reader announces extra information announced on dropdown.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8620]
## Action Performed:

1. Using Android Talkback + Chrome browser, open the URL staging.new.expensify.com/r/85021985        
2. Open settings and navigate to the preferences using Swipe Gesture and Double Tap.                                                    
3. New content will appear, navigate to the dropdown using Swipe Gesture and Double Tap.   
4. Listen to the screen reader and observe the issue.

## Expected Result:
Only the information of a dropdown should be announced after expanding/collapsing. Screen should not announce anything without user input. 

## Actual Result:
Screen reader announce extra information of the dropdown after expanding/collapsing, which is not visually present on the page. Screen reader announced as "Chrome focus".

**Other Occurrences:**
>Same issue repro on PR #8436, #8969, #8692, #8897, #8765

## Workaround:
Yes

## Area issue was found in:
Settings -> Preferences

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users might be confused by the announcement of more information and about where the current focus.

## Suggested resolution:
Make sure the screen reader announce only the information of element visually present on the page.                                                                           Refer to:              
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

## Platform:

- Android
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

https://user-images.githubusercontent.com/97600950/163125912-b5b2924c-d28b-43dc-a325-40af79c5499b.mp4
